_docs/setup/kubernetes/mesh-expansion.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Mesh Expansion
+overview: Instructions to add external machines and expanding Istio.
+
+order: 60
+
+layout: docs
+type: markdown
+---
+
+Instructions to configure Istio on a Kubernetes Cluster so it can be expanded with
+services running on cloud or on-prem VMs or external machines.
+
+## Prerequisites
+
+* You have [installed Istio](quick-start.html) on Kubernetes.
```

Comment:
To maintain consistency w/ other docs: "Setup Istio on Kubernetes by following the instructions in the [Installation guide](quick-start.html)."

Reply:
ok (btw "installation guide" or "Installation guide" or "Installation Guide" ?)

Reply:
Should be all lowercase: installation guide. I'll try to change that in other docs as I see it.